src/Sulu/Bundle/WebsiteBundle/Resources/config/exception_controller.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sulu_website.exception.controller.class">Sulu\Bundle\WebsiteBundle\Controller\ExceptionController</parameter>
```

Comment:
Didn't we stop using parameters like these?

Reply:
I just moved it. It did exist before and don't wanted to remove it.

Reply:
Ah, sorry, thought for a second we can do BC breaks now, but that's actually not true :sweat_smile: